base/docs/helpdb/Base.jl

```diff
@@ -10087,9 +10091,10 @@ no effect outside of compilation.
 include_dependency
 
 """
-    randn!([rng], A::Array{Float64,N})
+    randn!([rng], A::Array{F,N})
 
 Fill the array `A` with normally-distributed (mean 0, standard deviation 1) random numbers.
+The type `F` can be `Float16`, `Float32` or `Float64`.
 Also see the rand function.
```

Comment:
Done. As much as I like this information, I found a bit strange to find this cross-reference here but not in most of other rand-related functions. I would love to have a standard convention in docstring for "see also" references (a la man-pages).